### PR TITLE
Fixed issue #13910: Unable to use saveall button (an other button)

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: for default action : see https://github.com/LimeSurvey/LimeSurvey/blob/ed9fc2b6455b0c09c6672dcd1a1d130139838e6b/themes/survey/vanilla/views/subviews/header/start_form.twig#L71
Dev: then same behaviour with and without js

### CODE REVIEW

**Comment by Shnoulle** [comment]
Issue here https://bugs.limesurvey.org/view.php?id=13860
Answer is «advanced customization» … really ?

File: themes/survey/vanilla/scripts/ajaxify.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ var AjaxSubmitObject = function () {
         // Restrict to [type=submit]:not([data-confirmedby])
         // - :submit is the default if button don't have type (reset button on slider for example),
         // - confirmedby have their own javascript system
-        $(document).on('click', '#ls-button-submit, #ls-button-previous', function (e) {
+        $(document).on('click', '#limesurvey [type=submit]:not([data-confirmedby]:not(disabled))', function (e) {
```
Comment (lacrioque) [inline themes/survey/vanilla/scripts/ajaxify.js]: Why you are so keen on having this selector? I still strongly oppose this.
It is not specific enough and the effects of pagetriggers should always only be done by page triggers.
One way around it is to add a specific selector class just for this kind of purpose? Would you be ok with that?
It is a simple change, just use i.g. .selector__trigger-page-submit to select the triggering buttons.

Comment (Shnoulle) [reply themes/survey/vanilla/scripts/ajaxify.js]: A data , not a class. data-action='submit' for example. But i really think the purpose (action='submit') of a button must launch submit action … this action was done without js, then same action must be done with js …

Currently : something working on 2.73 + answer_html + 2.6lts didn't work in master. 

And the current system for disable multiple submit didn't work : https://bugs.limesurvey.org/view.php?id=13911

**Comment by Shnoulle** [comment]
Javascript must add feature, not remove feature …

If an action can be done without javascript : it mus can be done with javascript. But an action doable only with javascript can be OK (doable without js is better, but not mandatory).

File: themes/survey/vanilla/scripts/ajaxify.js (hunk 1 above)
Comment (Shnoulle) [reply themes/survey/vanilla/scripts/ajaxify.js]: Yes, and … deactivate ajaxmode : click on Index button : it work. Activate ajaxmode, click on index button : it don't work …

**Comment by LouisGac** [comment]
I didn't follow all your debate about the selector, but remember this: it's possible to create a theme without ajax and just using the EM js. I did it before 3.0 release. I will redo it and add it to premium theme.

**Comment by Shnoulle** [comment]
Yes @LouisGac : i know : i can redo a template from scratch, like i do getSleleton and fix a lot of part in it. But still : i think LimeSurvey core must offer the best user experience by default …

**Comment by LouisGac** [comment]
sure.

**Comment by Shnoulle** [comment]
OK, then we leave broken system in LS core …

**Pull request closed without merging**